build-new-version.sh

```diff
@@ -275,7 +286,7 @@ EOF
     }
     return get_by_triple(URLS, triple)
 
-DEPENDS += "rust-bin-cross-\${TARGET_ARCH} (= ${TARGET_VERSION})"
+DEPENDS += "rust-bin-cross-\${TARGET_ARCH} (= ${TARGET_VERSION}-${NIGHTLY_DATE})"
```

Comment:
This looks like it would break generation for the non-nightly case (trailing dash).

Reply:
Yes you are right. Thanks